src/mlpack/methods/cf/cf_model_impl.hpp

```diff
@@ -105,15 +118,34 @@ void CFModel::Train(const MatType& data,
                     const size_t rank,
                     const size_t maxIterations,
                     const double minResidue,
-                    const bool mit)
+                    const bool mit,
+                    const std::string& normalization)
 {
   // Delete the current CFType object, if there is one.
   boost::apply_visitor(DeleteVisitor(), cf);
 
   // Instantiate a new CFType object.
   DecompositionPolicy decomposition;
-  cf = new CFType<DecompositionPolicy>(data, decomposition,
-      numUsersForSimilarity, rank, maxIterations, minResidue, mit);
+  if (normalization == "overallMean")
```

Comment:
Do you mind to use `{}` here around the if/else cases to improve the readability?